Explain color highlighting

swellander opened the pull request:
Closes #53 

<img width="1603" height="449" alt="image" src="https://github.com/user-attachments/assets/993d0070-afa1-4304-91ba-cee8f36cae74" />

### Changes
- Moves the Staking tab table tooltip (say that 5 times fast!) next to the "Oracle Nodes" header. Does the same on the Whitelist tab
- Adds 3 tooltips to describe the meaning of the color highlighting that happens while running the simulation script
- Turns the price report cell gray when stale on the Staking tab

AzimovS reviewed (changes requested):
Hi, is there any reason for making it on the right side?
In the Assertion page, now it is cutted
![Image](https://github.com/user-attachments/assets/5a6ea599-486b-4c0c-8a42-47a15972b3fc)

What do you think, do we need it for the Last Reported Price (USD) in the Whitelist table? 
<img width="286" height="523" alt="Image" src="https://github.com/user-attachments/assets/eaa0c053-071f-46bc-8bb2-5345f56d9229" />

We also highlight it in the PriceWidget component. In your opinion, is it important to add it here as well?

<img width="573" height="135" alt="Image" src="https://github.com/user-attachments/assets/d7054ae1-52a2-4af2-ba1b-009cffa0882a" />

I just noticed that PriceWidget doesn't work correctly. Please let me know, if you would like to change this within this PR or should I make a separate PR for this. I think it should be changed to something like this:
`
const getHighlightColor = (oldPrice: bigint | undefined, newPrice: bigint | undefined): string => {
  if (oldPrice === undefined || newPrice === undefined) return "";

  const change = Math.abs(parseFloat(formatEther(newPrice)) - parseFloat(formatEther(oldPrice)));
  if (change < 3) return "bg-success";
  if (change < 10) return "bg-warning";
  return "bg-error";
};
`

swellander commented:
@AzimovS thanks for the review!

I think I've addressed all your comments. Good catch with the PriceWidget. I ended up increasing the color ranges so the user will see more variety.

escottalexander reviewed (changes requested):
Seeing this on the whitelist and staking page
<img width="540" height="322" alt="image" src="https://github.com/user-attachments/assets/2ca7a886-e649-4196-8bfd-b33f0e396848" />

swellander commented:
@escottalexander thanks for catching that. They should all be positioned correctly now

swellander commented:
@AzimovS thanks I think that's a good call, sorry I missed that on my last pass.
